packages/enzyme-test-suite/test/RSTTraversal-spec.jsx

```diff
@@ -155,13 +153,6 @@ describe('RSTTraversal', () => {
       expect(nodeHasProperty(<div foo={-Infinity} />, 'foo', '-Infinity')).to.equal(true);
       expect(nodeHasProperty(<div foo={0} />, 'foo', '-Infinity')).to.equal(false);
     });
-
-    it('should throw when un unquoted string is passed in', () => {
```

Comment:
it'd be even better if we could make a new package that avoids forking and instead directly depends on scalpel, but i'm not sure how hard that'd be :-/